cijoe/scripts/fio_latency.py

```diff
@@ -116,12 +120,20 @@ def prepare(self) -> None:
 
 class KernelEngine(Engine):
     def prepare(self) -> None:
-        self.cijoe.run(" ".join(["xnvme-driver", "reset"]))
+        shell = (
+            self.cijoe.config.options.get("cijoe", {}).get("run", {}).get("shell", "sh")
+        )
+        if shell != "pwsh":
+            self.cijoe.run(" ".join(["xnvme-driver", "reset"]))
```

Comment:
This brings up an interesting point.
`xnvme-driver` is really only relevant on Linux and FreeBSD. Maybe we shouldn't even install it on macOS and Windows.
If we eventually get userspace drivers on either macOS or Windows, then `xnvme-driver` should probably be expanded to support attaching/detaching the devices on those as well.

Reply:
the problem is exactly that `xnvme-driver` is not installed on windows (because it is not relevant), and therefore it gives the following error, making the whole workflow fail.
`The term 'xnvme-driver' is not recognized as the name of a cmdlet, function, script file, or operable program.`

so i had to make sure it is not run on windows to begin with